suffuse/util.go

```diff
@@ -97,3 +97,9 @@ func ScratchFile() Path {
   deleteOnExit = append(deleteOnExit, path)
   return path
 }
+
+// https://github.com/golang/go/issues/1312
+func FileExists(path string) bool {
+  _, err := os.Stat(path)
```

Comment:
Maybe path.FileExists() is Stat and path.LinkExists() is Lstat. This issue pervades everything in fact, and we should try to find a good scheme for making it clear when symbolic links will be followed. It's especially important that it do the sensible thing when it exists. The standard approach has been to pollute method calls in all kinds of different ways with config options like NOFOLLOW. It's awful.